docs/guides/connecting_to_your_data/datasource_configuration/sql_components/_part_data_connector_required_keys_overview.mdx

```diff
@@ -0,0 +1,29 @@
+<!-- props.data_connector_type: string value of the data connector's class name. -->
+<!-- props.data_connector_name: string value of the name assigned to the data connector in previous examples. -->
+<!-- props.inferred: boolean value that determines if the base key list should be `class_name`, `splitter_method`, `splitter_kwargs`, `introspection_directives` (true) -->
+<!-- props.configured: boolean value that determines if the base key list should be `class_name`, `assets` (true) -->
+<!-- props.runtime: boolean value that determines if the base key list should be `class_name`, `batch_identifiers` (true) -->
+
+<div>
+    <p>
+        When defining an <code>{props.data_connector_type}</code> you will need to provide values for { props.inferred && <span>one required key</span> || <span>two required keys</span> } in the Data Connector's configuration dictionary (the currently empty dictionary that corresponds to <code>"{props.data_connector_name}"</code> in the example above).  { props.inferred && <span>This key/value pair consists of:</span> || <span>These key/value pairs consist of:</span> }
```

Comment:
```suggestion
        When defining a(n) <code>{props.data_connector_type}</code> you will need to provide values for { props.inferred && <span>one required key</span> || <span>two required keys</span> } in the Data Connector's configuration dictionary (the currently empty dictionary that corresponds to <code>"{props.data_connector_name}"</code> in the example above).  { props.inferred && <span>This key/value pair consists of:</span> || <span>These key/value pairs consist of:</span> }
```

Reply:
I implemented a different correction so that a/an would change appropriately, instead of having them combined as a(n).